<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commodore64</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="generatePB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PxPlus IBM BIOS-2y</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PxPlus IBM BIOS-2y</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/Images/assets/images/perceptron.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotField" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="kirbo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PxPlus IBM BIOS-2y</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="yuri">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>140</width>
      <height>290</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>290</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="runPB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PxPlus IBM BIOS-2y</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="epochsLE">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="epochsLB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PxPlus IBM BIOS-2y</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Epochs</string>
    </property>
   </widget>
   <widget class="QLabel" name="ethaLB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PxPlus IBM BIOS-2y</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Etha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ethaLE">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>generatePB</zorder>
   <zorder>label_4</zorder>
   <zorder>plotField</zorder>
   <zorder>kirbo</zorder>
   <zorder>resetPB</zorder>
   <zorder>yuri</zorder>
   <zorder>runPB</zorder>
   <zorder>epochsLE</zorder>
   <zorder>epochsLB</zorder>
   <zorder>ethaLB</zorder>
   <zorder>ethaLE</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
